app/services/mailchimp/bot.rb

```diff
@@ -65,7 +65,7 @@ def resubscribe_to_newsletter
     end
 
     def manage_community_moderator_list
-      return false unless user.has_role?(:trusted)
+      return false unless user.has_role?(:trusted) || SiteConfig.mailchimp_community_moderators_id.blank?
```

Comment:
These guard clauses might be overkill, but it seemed to make sense to me -- don't check if we don't have the newsletter enabled.

Reply:
`has_role?` accesses the DB, so if you add this extra guard, I think you should add it before the role check, not after.